### PR TITLE
Update to webpack 2 and React 15

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

File: app/app.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8">
     <title>Tempora</title>
-    <link rel="stylesheet" href="../node_modules/font-awesome/css/font-awesome.min.css"/>
+    <link rel="stylesheet" href="../node_modules/font-awesome/css/font-awesome.min.css" />
```
Comment (jasonlemay) [inline app/app.html]: rm space (or fix 2 links below so it's the same)

File: app/components/dashboard/Dashboard.js [hunk 2]
```diff
@@ -26,28 +27,33 @@ export default class Dashboard extends Component {
...
-                            <a href="#" onClick={() => this.props.orderByName(entries)}>
-                                <i className="fa fa-sort-alpha-asc" aria-hidden="true"></i>
-                            </a>
-                        </td>
-                    </tr>
-                    {this.showEntries(entries)}
+                <tr className="dashboard__content__sep">
+                    <td colSpan="2">{entriesTitle}</td>
+                    <td>
+                        <a href="javascript:;" onClick={() => this.props.orderByTime(entries)}>
```
Comment (jasonlemay) [inline app/components/dashboard/Dashboard.js:35]: todo: anchor should go to a custom url, if not, it should be a button

File: .eslintrc [hunk 3]
```diff
@@ -22,8 +21,17 @@
                 ]
             }
         ],
+        "indent": [
```
Comment (Chris-V) [inline .eslintrc]: I will create a NPM package for out lint configs, the copy-paste is getting old ;)

Comment (Chris-V) [inline .eslintrc:24]: I will create a NPM package for our lint configs. The copy paste is getting old.

File: app/style.scss [hunk 4]
```diff
@@ -1,33 +1,24 @@
 /**
  * -- SETTINGS
  */
-
```
Comment (jasonlemay) [inline app/style.scss]: Revert that file. Don't autoformat on .scss

File: app/components/issues/Issues.js [hunk 5]
```diff
@@ -78,25 +76,34 @@ export default class Issues extends Component {
...
+                    className="page__item__link page__item__link--with-icon"
+                >{issue.key}<span className="page__item__link__desc">{issue.fields.summary}</span>
                 </Link>
-                <a href="#" className="page__item__link-icon" onClick={() => shell.openExternal(ISSUE_BASE_URL + "/" + issue.key)}>
-                    <i className="fa fa-external-link" aria-hidden="true"></i>
+                <a
+                    href="javascript:;"
+                    className="page__item__link-icon"
+                    onClick={() => shell.openExternal(StorageService.getInstanceURL() + '/browse/' + issue.key)}
+                >
```
Comment (Chris-V) [inline app/components/issues/Issues.js:105]: We got some more cleanup to do here I guess

File: package.json [hunk 6]
```diff
@@ -27,84 +27,88 @@
     "electron": "./node_modules/.bin/electron"
   },
   "devDependencies": {
-    "asar": "^0.12.2",
-    "babel-core": "^6.14.0",
-    "babel-eslint": "^7.0.0",
-    "babel-loader": "^6.2.5",
+    "asar": "^0.13.0",
```
Comment (Chris-V) [inline package.json:30]: What about Yarn since we moved our other projects to it.

**Review by Chris-V — APPROVED** [review]

**Comment by jasonlemay** [comment]
@meriouma I guess you can merge it?

**Pull request merged**